mirgecom/multiphysics/thermally_coupled_fluid_wall.py

```diff
@@ -732,62 +733,58 @@ def grad_cv_bc(
             self, dcoll, dd_bdry, gas_model, state_minus, state_bc, grad_cv_minus,
             **kwargs):
         """
-        Return external grad(CV) used in the boundary calculation of viscous flux.
-
-        Specify the velocity gradients on the external state to ensure zero
-        energy and momentum flux due to shear stresses.
+        Return grad(CV) to be used in the boundary calculation of viscous flux.
 
         Gradients of species mass fractions are set to zero in the normal direction
         to ensure zero flux of species across the boundary.
         """
         dd_bdry = as_dofdesc(dd_bdry)
         normal = state_minus.array_context.thaw(dcoll.normal(dd_bdry))
 
-        grad_mom_bc = self._slip.grad_momentum_bc(
-            state_minus, state_bc, grad_cv_minus, normal)
-
         grad_species_mass_bc = self._impermeable.grad_species_mass_bc(
             state_minus, grad_cv_minus, normal)
 
         return make_conserved(
             grad_cv_minus.dim,
             mass=grad_cv_minus.mass,
             energy=grad_cv_minus.energy,
-            momentum=grad_mom_bc,
+            momentum=grad_cv_minus.momentum,
             species_mass=grad_species_mass_bc)
 
     def temperature_plus(self, dcoll, dd_bdry, gas_model, state_minus, **kwargs):
-        """Get the exterior T on the boundary."""
-        kappa_minus = (
-            # Make sure it has an array context
-            state_minus.tv.thermal_conductivity + 0*state_minus.mass_density)
-        return self._thermally_coupled.temperature_plus(
-            dcoll, dd_bdry, kappa_minus, state_minus.temperature)
+        """Get T on the external/plus side (aka wall)."""
+        return _project_from_base(dcoll, dd_bdry, self._t_plus)
 
-    def grad_temperature_bc(
-            self, dcoll, dd_bdry, gas_model, state_minus, grad_cv_minus,
-            grad_t_minus, **kwargs):
-        """Get grad(T) on the boundary."""
-        return self._thermally_coupled.grad_temperature_bc(
-            dcoll, dd_bdry, grad_t_minus)
+    def grad_temperature_bc(self, dcoll, dd_bdry, gas_model, state_minus,
+                            grad_cv_minus, grad_t_minus, **kwargs):
+        """Get grad(T) on the boundary uses the internal/minus value (aka fluid).
 
+        This is the same logic of an isothermal BC.
+        """
+        return _project_from_base(dcoll, dd_bdry, grad_t_minus)
```

Comment:
I think we can just omit this function like `IsothermalWallBoundary` does. (`PrescribedFluidBoundary` defaults to this behavior.)